researcher_ids/README.md

```diff
@@ -0,0 +1,89 @@
+        2. Access to some datasets may require a specific ControlledAccessGrants claim to be present.
+    4. The claims are extracted from the Passport and the contents of the claims are compared to the required claims for the access policy. The Claim Clearinghouse will accept or reject claims and tries to find a set of acceptable claims that match the access policy.  
+        1. The "[expires](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE/edit#heading=h.tdge5487ert9)" field is checked to make sure it hasn't expired, and may use [special expiry checking logic](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE/edit#heading=h.ruipgcz7pe27) to make sure it isn't going to expire soon.
+        2. The "[source](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE/edit#heading=h.7gkkcdjjulcj)" field is checked to see if it is from a trusted source from a trusted source whitelist.
+        3. The "[value](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE/edit#heading=h.vddhgptnqj4f)" field is checked to see if it meets the requirements of the access policy.
+        4. The "[asserted](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE/edit#heading=h.x2qu20cktltz)" and "[by](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE/edit#heading=h.bjnftdj471t6)" fields may be checked as well, depending on the policy.
+4. If the Claim Clearinghouse decides that the Passport meets the access policy for the data in question, the service proceeds to authorize the researcher's use of the data.
+    1. Some Claim Clearinghouses will authorize use of the data by changing the researcher's permissions and issuing a cloud-specific access token to read the bytes using their own cloud-native tools and services.
+    2. Other Claim Clearinghouses will read the bytes from another service using special access token that it holds which contains permission to do so, then return the bytes back to the researcher.
+    3. Yet other varieties of Claim Clearinghouses will just proceed taking an action internally or reading bytes directly from a database in and of itself without much need for downstream access tokens.
+5. Some systems will use the Passport once, exchange it for a downstream access token, and not use the Passport again. Other systems will use the Passport multiple times until the passport token expires.
+    1. Some systems can issue refresh tokens that can be exchanged for new Passport tokens with a fresh set of claims and updated expiry timestamp.
+    2. Other systems do not issue refresh tokens, and the only option to get a new Passport is to have the user do a fresh login (OIDC Authorization Flow) to fetch a new Passport.
+
+## Flow Of Claims
+
+![alt text](https://github.com/ga4gh-duri/flow_of_claims.png "Flow of Claims")
```

Comment:
The PNG file is missing?

Reply:
Also it may be worth putting image assets in the /assets/img/ directory.

Reply:
I've changed the path to /assets/img/  --- I'd don't have push access to the repository to add the image in that folder however. Can you try it? The image should be named "flow_of_claims.png"